src/components/Dashboard/FilingHistoryList/LimitedRestorationExtensionFiling.vue

```diff
@@ -7,10 +7,10 @@
 
     <p>
       The period of restoration was successfuly extended and is active
-      <strong>until {{ expiryDateFriendly }}</strong>. At the end of the extended limited
-      restoration period, the company will be automatically dissolved. If you require assistance
-      to extend a limited restoration/reinstatement or wish to convert your restoration from a
-      limited period to a full restoration, please contact BC Registries staff:
+      <strong>until {{ expiryDateFriendly || '[Unknown]' }}</strong>. At the end of the extended
```

Comment:
I think it's better for the script to handle fallback strings. That way, the script can handle the "computations" and the script can focus on language, etc.